contracts/staking/StakingRewardsUpgradeable.sol

```diff
@@ -64,6 +66,7 @@ contract StakingRewardsUpgradeable is
         rewardsToken = IERC20(_rewardsToken);
         stakingToken = IERC20(_stakingToken);
         rewardsDistribution = _rewardsDistribution;
+        minimumStakeAmount = _minimumStakeAmount;
```

Comment:
As a note, see https://github.com/CryptoBlades/cryptoblades/pull/389#discussion_r675188791. Again, won't actually enforce it for now, so it's just something to keep in mind for the future.